ui/backend/server/trackingserver_projects/api.py

```diff
@@ -125,7 +125,7 @@ async def get_project_by_id(
     try:
         project = await Project.objects.aget(id=project_id)
     except Project.DoesNotExist:
-        raise HttpError(404, f"Could not find project with ID: {project_id}")
+        raise HttpError(404, f"Could not find project with ID: {project_id}") from None
```

Comment:
Why not `as e...from e`? Do you want to detach the errors on purpose? Same question for `ui/backend/server/trackingserver_run_tracking/api.py`

Reply:
yep. User doesn't need to see internal stack trace I don't think.